docs/organizations/verify-user-permissions.mdx

```diff
@@ -48,9 +48,11 @@ The examples below work for both SSR and CSR. Examples are written for Next.js A
   </Tab>
 
   <Tab>
-    The `useAuth()` hook can help you access what permissions a user has by using `has()`. In the example below, `has({ permission: "org:team_settings:manage" })` returns `false` if the user does not have the `org:team_settings:manage` permission.
+    Use the [`useAuth()`](/docs/references/react/use-auth) hook to access the `has()` helper in Client Components.
 
-    ```tsx filename="/app/dashboard/settings/form.tsx"
+    The following example uses `has()` to inspect a user's permissions granularly. If the user doesn't have the `org:team_settings:manage` permission, `has()` returns `false`, causing the component to return `null` instead of rendering its children.
```

Comment:
IMO it's not entirely necessary to describe what these examples are doing in detail like this, as they can understand all that from reading the code. I'd personally prefer a high-level description of what the example's goal is, like "The following example renders a UI **only if** the user has the necessary permissions." or something like that